File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseRoutesResult.scala

package org.openapitools.client.model


case class ResponseRoutesResult (
    _searchId: String,
    _locations: List[ResponseRoutesLocation],
    _unreachable: List[String]
)
object ResponseRoutesResult {
    def toStringBody(var_searchId: Object, var_locations: Object, var_unreachable: Object) =
        s"""
        | {
        | "searchId":$var_searchId,"locations":$var_locations,"unreachable":$var_unreachable
        | }
        """.stripMargin
}

File: dart-jaguar/lib/model/response_routes_result.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


import 'package:openapi/model/response_routes_location.dart';

part 'response_routes_result.jser.dart';

class ResponseRoutesResult {
  
  @Alias('search_id', isNullable: false,  )
  final String searchId;
  
  @Alias('locations', isNullable: false,  )
  final List<ResponseRoutesLocation> locations;
  
  @Alias('unreachable', isNullable: false,  )
  final List<String> unreachable;
  

  ResponseRoutesResult(
      

{
    
     this.searchId = null,  
     this.locations = const [],  
     this.unreachable = const [] 
    }
  );

  @override
  String toString() {
    return 'ResponseRoutesResult[searchId=$searchId, locations=$locations, unreachable=$unreachable, ]';
  }
}

@GenSerializer(nullableFields: true)
class ResponseRoutesResultSerializer extends Serializer<ResponseRoutesResult> with _$ResponseRoutesResultSerializer {

}

